build.gradle.kts

```diff
@@ -0,0 +1,117 @@
+import com.github.jengelman.gradle.plugins.shadow.tasks.ShadowJar
+import org.jetbrains.kotlin.gradle.tasks.KotlinCompile
+import pl.allegro.tech.build.axion.release.domain.hooks.HookContext
+import pl.allegro.tech.build.axion.release.domain.hooks.HooksConfig
+import java.time.OffsetDateTime
+import java.time.ZoneOffset
+import java.time.format.DateTimeFormatter
+
+plugins {
+    kotlin("jvm") version "1.3.61"
+    id("com.github.johnrengelman.shadow") version "5.2.0"
+    id("pl.allegro.tech.build.axion-release") version "1.10.3"
+    id("org.jlleitschuh.gradle.ktlint") version "9.1.1"
+}
+
+val repoRef = "SimpleMC\\/SimpleHealthbars2"
```

Comment:
SimpleHealthBars2 reference

Reply:
I took the time to pull it out into a val so I could change it and just didn't...